### ISSUE TITLE
intel 19.0.4 AVX512 error

### ISSUE THREAD

**Issue by tjhei** [dealii/dealii#8371]
I am getting
```
/scratch1/02578/theister/deal-git/include/deal.II/base/vectorization.h(1355): error: a value of type "dealii::VectorizedArray<float, 16>" cannot be assigned to an entity of type "const __m512 &"
      t3 = out[0];
```

**Comment by tjhei** [comment]
@kronbichler do we need to do a load using instrinsics?

**Comment by peterrum** [comment]
I have fixed this as part of #8348.

**Comment by tjhei** [comment]
Did you discuss with @kronbichler ? I thought ``= {}`` causes a bug in gcc.

**Comment by peterrum** [comment]
@tjhei: you are completely right! I forgot that. I guess @kronbichler wanted to write:
```cpp
  __m512 t0, t1, t2, t3;
  if (n_chunks > 0)
    t3 = out[0].data;
```
@kronbichler's test program was working directly on `__m512*` and not on on `VectorizedArray<float>`.

**Comment by peterrum** [comment]
@tjhei: I would update  #8348 accordingly. Or should it be a separate PR?

**Comment by kronbichler** [comment]
@peterrum thinking about, let us set up a separate PR so it can get in quickly.

**Comment by peterrum** [comment]
I'll do it right away.

**Comment by tjhei** [comment]
yep, that would be great!

**Comment by tjhei** [comment]
closed via #8373
